fixtures/chauvet-dj/120st.json

```diff
@@ -0,0 +1,168 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Follow Spot 120ST",
+  "shortName": "120ST",
+  "categories": ["Dimmer", "Other"],
```

Comment:
Not a Dimmer, though I don't think we have a great category for these -- just Other might be the best we can do right now.

Reply:
Dimmer Category removed